### DISCUSSION TITLE
Libvvenc support for adding stream and subtitles?

### DISCUSSION THREAD

**Discussion by PenDZ-Alter** [fraunhoferhhi/vvenc#317] [Q&A] [answered]
Hello,
Does libvvenc encoder support adding stream and subtitles?

Because, when i adding more stream video in ffmpeg it return error.

**Comment by adamjw24** [comment, marked as answer]
VVenC is a raw video encoder. Adding additional streams happens at the container level, which is the domain of the packager. In that sense, you having looked at ffmpeg is already a good start, since it is also a fairly capable muxer.

The problem you are having is not related to VVenC. Please check with the developers of your muxer, in this case ffmpeg, about what the problem might be.

Best, Adam
